ontology/uco/observable/observable.ttl

```diff
@@ -6579,6 +7032,28 @@ observable:ValuesEnumeratedEffectFacet
 	sh:targetClass observable:ValuesEnumeratedEffectFacet ;
 	.
 
+observable:VideoCamera
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:Device ;
+	rdfs:label "VideoCamera"@en-US ;
+	rdfs:comment "A video camera is a camera that captures video and audio in digital memory."@en-US ;
```

Comment:
Considering CCTV systems, we might want to leave 'audio' out of scope of this concept.

Reply:
And a video camera does not necessarily capture anything in digital memory.
For that it seems like there would need to be a DigitalRecordingVideoCamera subclass of a DigitalVideoCamera subclass of VideoCamera.
Some video cameras are analog and some only stream and do not record.

Reply:
We will wait until after 1.0.0 to integrate this one.  Cutting for now.

Reply:
Cut in [1316130](https://github.com/ucoProject/UCO/commit/1316130).